content/hardware/07.opta/opta-family/opta/tutorials/05.getting-started-with-aws-iot-core/content.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Getting Started with AWS IoT Core'
+title: 'Connect Arduino OPTA to AWS IoT Core'
```

Comment:
```suggestion
title: 'Connect Arduino Opta™ to AWS IoT Core'
```

Reply:
@jhansson-ard the suggestion makes sense, accepted to move this forward